test/app/JoinServer.test.js

```diff
@@ -42,7 +42,16 @@ describe('POST /join', async () => {
 				address: '0x766760C748bcEcf5876a6469a6aed3C642CdA261',
 				request: JSON.stringify({
 					dataUnion: '0x81ed645D344cB2096aBA56B94d336E6dcF80f6C6',
-					chain: 'test-chain',
+					chain: 'polygon',
+				}),
+			},
+		},
+		{
+			name: 'send join data union request without chain',
```

Comment:
This case should fail - the join request shouldn't be valid without the `chain` field